monkestation/code/modules/antagonists/mimic/mimic.dm

```diff
@@ -2,23 +2,24 @@
 #define MIMIC_DISGUISE_COOLDOWN 5 SECONDS
 
 #define REPLICATION_COST(mimic_count) (1 + round((mimic_count / 2)))
+#define EVOLUTION_COST(mimic_count, multiplier) round(multiplier*REPLICATION_COST(mimic_count))
 
 /mob/living/simple_animal/hostile/alien_mimic
 	name = "mimic"
 	real_name = "mimic"
 	desc = "A morphing mass of black gooey tendrils."
 	speak_emote = list("warbles")
 	emote_hear = list("warbles")
-	faction = list("aliens") //don't wanna have them attack eachother
-	icon = 'monkestation/icons/mob/animal.dmi'
+	faction = list("aliens")
+	icon = 'monkestation/icons/mob/mimics.dmi'
 	icon_state = "mimic"
 	icon_living = "mimic"
 	icon_dead = "mimic_dead"
 	move_to_delay = 0.5 SECONDS
-	var/disguised_move_delay = 0.4 SECONDS
-	var/undisguised_move_delay = 0.05 SECONDS
+	var/disguised_move_delay = 0.2 SECONDS
```

Comment:
this isn't a typical review comment buts its being left here because i don't know about halving the movement delay

Reply:
I just tested, and compared to a t3 mimic (same speed as it was originally), it only ends up getting across the top hall on pubby station maybe 1-2 seconds faster